src/euroeval/dataset_configs/estonian.py

```diff
@@ -13,3 +13,20 @@
     task=SENT,
     languages=[ET],
 )
+
+WINOGRANDE_ET_CONFIG = DatasetConfig(
+    name="winogrande_et",
+    pretty_name="the Estonian Winogrande dataset",
+    huggingface_id="EuroEval/winogrande_et",
+    task=COMMON_SENSE,
+    languages=[ET],
+    # requires custom templates as WinoGrande is different from
+    # the usual multiple choice tasks
+    _prompt_prefix="Sulle esitatakse lüngaga (_) tekstülesanne "
+    "ja kaks vastusevarianti (a ja b).\n",
```

Comment:
Is it true that this means "You are presented with a text problem with a gap (_) and two possible answers (a and b).\n"?

This prefix is only used when evaluating base models, which is run in a few-shot setting. So we need to set the prefix in a plural form, akin to "The following are text problems with a gap (_) and two possible answers (a and b)."

This maintains consistency with other multiple-choice datasets. I removed the newline as well, as this is added automatically.

Reply:
Makes sense. I've updated the phrasing here.